Validation/CTPPS/python/year_2016/randomXiThetaGunProducer_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@jan-kaspar @alberto-sanchez why this doesn't go in Configuration/Generator? The PR history is so long that this can be done afterwards, but still this doesn't seem to me the best place for this fragment

Reply:
This file is used only for internal validation of (CT)PPS code, that's why it is "hidden" in the CTPPS validation package. On the other hand, I agree that it could be of general interest to have a "reference" config for gun-like simulation of PPS protons. I've added this task to our TODO list https://github.com/CTPPS/cmssw/issues/40.